en/docs/design/api-security/oauth2/grant-types/authorization-code-grant.md

```diff
@@ -69,7 +69,7 @@ The steps below show how access tokens are generated for the authorization code
 !!! note
     **Before you begin**
 
-    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/5.10.0/learn/deploying-the-sample-app/#deploying-the-playground2-webapp) .
+    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-playground2-webapp).
```

Comment:
```suggestion
    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/5.10.0/learn/deploying-the-sample-app/#deploying-the-playground2-webapp).
```

Reply:
@samgnaniah it is not correct to have the link as latest here. 5.10.0 is correct. If u see in the git issue it says that it has been fixed for 400 already

Reply:
I was going by the Release Matrix here [1]. Is this an older sample version we're using?

[1] - https://wso2.com/products/carbon/release-matrix/

Reply:
Sorry, my bad. APIM 4.0.0 is compatible with IS 5.11.0. Let's add the version in the link without just pointing to latest/